contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsBilling.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title Chainlink Functions billing subscription registry interface.
+ */
+interface IFunctionsBilling {
+  struct RequestBilling {
+    // a unique subscription ID allocated by billing system,
+    uint64 subscriptionId;
+    // the client contract that initiated the request to the DON
+    // to use the subscription it must be added as a consumer on the subscription
+    address client;
+    // customer specified gas limit for the fulfillment callback
+    uint32 gasLimit;
+    // the expected gas price used to execute the transaction
+    uint256 gasPrice;
```

Comment:
maybe expectedGasPrice is better here, right?  IIUC, then this is just used during the cost estimation and blocking funds during sendRequest.

Reply:
Yep, agreed